Assets/MixedRealityToolkit/InputModule/Scripts/Focus/SimpleSinglePointerSelector.cs

```diff
@@ -1,9 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using MixedRealityToolkit.InputModule.Cursor;
+using MixedRealityToolkit.InputModule.EventData;
+using MixedRealityToolkit.InputModule.Gaze;
```

Comment:
Do we really need namespaces for each of these?
I think bundling most things in the base InputModule namespace should be enough.

Reply:
My only issue is that it would clutter the InputModule namespace.  Having each in their own namespace, ensures you know what are available when discovering elements.
This way, even through intellisense, you can easily discover what Cursors, Events, etc are available.

Reply:
Ah, I see, that makes sense.